cfpb/cfgov-refresh
# Adding Webpack stats option to gulp build script build flow

## Changes

- Modified code to add Webpack stats option
- Modified code to rename config variables to be consistent with other files. 

## Testing

1. Run `gulp scripts` and ensure there are no errors.
2. Run `gulp scripts:ondemand` and ensure there are no errors.
3. Run `gulp scripts:ondemand --stats` and you should see some stats about each chunk. There are plenty available [here](https://webpack.js.org/configuration/stats/#stats), but I just chose some I thought would be relevant. 


## Checklist
